### PR TITLE
Post release bug fixes

### PR DESCRIPTION
**Pull request by offirgolan**
- [x] #2 isInvalid helper
- [x] #5 Confirmation rework. It should now be generated under it's own property. (@stefanpenner let me know what you think about this one. Check out the models/user.js file for an example)
- [x] #7 Added sample hbs in readme
- [x] Fixed validator test blueprint to work with future version of ember

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -554,3 +563,19 @@ An alias to the first message in the messages collection.
...
 get(user, 'validations.attrs.username.message')
 ```
+
+## Templating Example ##
+Accessing validation information in your templates is really simple. Oh and don't worry, we know the path is pretty long! There are plans to condense that in the roadmap ahead.
+
+```handlebars
+<form>
+  {{input value=model.username placeholder="Username"}}
+  {{#if model.validations.attrs.username.isInvalid}}
```
Comment (stefanpenner) [inline README.md]: a helper for: `(is-valid model 'username')` might be nice

Comment (offirgolan) [inline README.md]: we could also do `(message-for model 'username')`

Comment (offirgolan) [inline README.md]: or even `(v-get model 'username' 'isValid)`

Comment (blimmer) [inline README.md]: I really like the idea of the combined `v-get` helper. the whole `model.validations.attrs.field.prop` is long to type.

File: blueprints/validator/files/tests/unit/validators/__name__-test.js [hunk 2]
```diff
@@ -1,9 +1,8 @@
 import Ember from 'ember';
-import { moduleFor, test }
-from 'ember-qunit';
+import { moduleFor, test } from 'ember-qunit';
 
 moduleFor('validator:<%= dasherizedModuleName %>', 'Unit | Validator | <%= dasherizedModuleName %>', {
-  needs: ['validator:messages']
+  integration: true
```
Comment (blimmer) [inline blueprints/validator/files/tests/unit/validators/__name__-test.js]: If you're going to make this an integration test, you should move the blueprint to `blueprints/validators/files/tests/integration/validators/__name__-test.js` to match conventions - also should probably change the description above. That said, I don't know if it needs to be an integration test.

File: tests/dummy/app/models/user.js [hunk 3]
```diff
@@ -35,12 +35,13 @@ var Validations = buildValidations({
...
-      message: 'do not match',
-      attributeDescription: 'Email addresses'
-    }),
+    })
   ],
+  emailConfirmation: validator('confirmation', {
+    on: 'email',
+    message: 'do not match',
+    attributeDescription: 'Email addresses'
+  }),
```
Comment (blimmer) [inline tests/dummy/app/models/user.js]: Nice, I like this much more.

File: blueprints/validator/files/tests/unit/validators/__name__-test.js (hunk 2 above)
Comment (offirgolan) [inline blueprints/validator/files/tests/unit/validators/__name__-test.js]: We ran into some registry/container issues with ember-release and above. Converting if to an integration test helped create the container we needed for the test. If you believe this should be a unit test, I can keep as it was. Might just need some more testing though.

Comment (blimmer) [inline blueprints/validator/files/tests/unit/validators/__name__-test.js]: I definitely don't have a strong opinion on it, was just curious what necessitated the change. Integration tests are very helpful, though slightly slower.

Comment (offirgolan) [inline blueprints/validator/files/tests/unit/validators/__name__-test.js]: Gotcha yeah I agree. Let me change the folder name to `integration` for now. In the future if want to change it back to unit, then we can make the switch back.

File: README.md (hunk 1 above)
Comment (offirgolan) [inline README.md]: Moved convo to #12

File: blueprints/validator/files/tests/unit/validators/__name__-test.js (hunk 2 above)
Comment (stefanpenner) [inline blueprints/validator/files/tests/unit/validators/__name__-test.js]: > @stefanpenner thoughts on this?

i prefer strict tests, we were slightly rushed yesterday :P

Comment (offirgolan) [inline blueprints/validator/files/tests/unit/validators/__name__-test.js]: Alright. Let me do some more testing on this.

**Comment by offirgolan** [comment]
Hmmm seems like everything is passing. @stefanpenner the integration test issues could have been something from ember-try? Going to merge this and create a minor release.

**Pull request merged**